desktop/apps/auctions2/templates/index.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { withLayout } from 'desktop/components/react/main_layout'
+
+function Index (props) {
+  const {
+    Layout: { Header, Body, Footer },
+    Templates: { Meta }
+  } = props
+
+  return (
+    <div>
+      <Header>
+        <Meta />
+      </Header>
+      <Body>
+        hello!
+      </Body>
+      <Footer />
+    </div>
```

Comment:
Ok, that would be great, and makes sense and is indeed the issue. In the meantime would you mind reviewing the rest of the PR? I would like to continue moving forward into auctions2. This is mainly a quick cleanup around the previous one and if we'd like to explore your approach it would take place in a different PR.